test/provider_os_memory.cpp

```diff
@@ -91,7 +91,7 @@ TEST_F(test, provider_os_memory_create_MBIND_FAILED) {
     ASSERT_EQ(os_memory_provider, nullptr);
 }
 
-TEST_F(test, provider_os_memory_alloc_free) {
+TEST_F(test, provider_os_memory_alloc_free_4k_alignment_0_SUCCESS) {
```

Comment:
I think test success is the default case, so we don't need to add this information. However, if we decide to do so, we should rename all the tests in our repo.

Reply:
Done